README.md

```diff
@@ -1,4 +1,34 @@
 # ubiquity_motor
 [![Build Status](https://travis-ci.org/UbiquityRobotics/ubiquity_motor.svg?branch=indigo-devel)](https://travis-ci.org/UbiquityRobotics/ubiquity_motor)
 
-Package that provides a ROS interface for the motors in UbiquityRobotics robots
+Introduction
+
+This is a Package that provides a ROS interface for the motors in UbiquityRobotics robots
+
+Installation - From Binary
+
+This package may be installed from binaries for both x86 and ARM architectures. To install the ppa for this package use the following command:
+
+[insert ppa command here]
```

Comment:
Agree with @jim-v the ppa lines should be removed. They can be replaced by something like:

The packages are available from the OSRF build farm.  See the ROS installation instructions for more details http://wiki.ros.org/kinetic/Installation/Ubuntu.